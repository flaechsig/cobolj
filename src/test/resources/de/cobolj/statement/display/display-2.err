Ein DISPLAY-Befehl kann mehrere Operanden enthalten