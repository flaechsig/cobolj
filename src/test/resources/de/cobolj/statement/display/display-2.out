Hello world!World 2
